### ISSUE TITLE
Do you have a solution to this problem

### ISSUE THREAD

**Issue by susu-gsupl** [zhiruiluo/deep_feature_fusion_rumor_detection#2]
pytorch_lightning.utilities.exceptions.MisconfigurationException: The provided lr scheduler `LambdaLR` doesn't follow PyTorch's LRScheduler API. You should override the `LightningModu
le.lr_scheduler_step` hook with your own logic if you are using a custom LR scheduler.

**Comment by zhiruiluo** [comment]
Hi @susu-gsupl, I tried to reproduce your exception but I couldn't. Did you use ```from torch.optim.lr_scheduler import LambdaLR``` or other implementation? I will push an commit with updated ```requirements.txt``` and some changes for compatiability. Let me know whether that resolves your problem.
